lib/middlewares/query_scope_authorization.ex

```diff
@@ -84,7 +84,9 @@ defmodule Rajska.QueryScopeAuthorization do
     arguments_source = get_arguments_source!(resolution, scope)
 
     arg_fields
-    |> Enum.all?(& apply_scope_authorization(resolution, scope, arguments_source, &1, rule, optional))
+    |> Enum.map(& get_scoped_struct_field(arguments_source, &1, optional, resolution.definition.name))
+    |> Enum.filter(& !!&1)
```

Comment:
We only want to filter out `nil`s here, right? I think `Enum.reject(&is_nil/1)` is clearer in that case

Reply:
Yes, indeed